tests/sentry/sentry_metrics/test_ratelimiters.py

```diff
@@ -3,6 +3,8 @@
 from sentry.sentry_metrics.indexer.base import KeyCollection
 from sentry.sentry_metrics.indexer.ratelimiters import WritesLimiter
 
+INDEXER_DB = "test-db"
```

Comment:
I don't think I get this test. There are only 2 valid db's - Postgres and CloudSpanner. What's it trying to test with this fake one?

Reply:
oh yeah this is left over, need to remove it